addons/mail/static/src/models/composer/composer.js

```diff
@@ -33,6 +33,13 @@ function factory(dependencies) {
             return res;
         }
 
+        /**
+         * @override
+         */
+         _created() {
+            this._activeThread = this.messageInEditing ? this.messageInEditing.originThread : this.thread;
```

Comment:
I would prefer this being a model field rather than a JS object property, especially when it refers to a model record.

Reply:
From the look of it, it's pretty easy to turn this into field.